facebook/hhvm
# Add VS Code settings

This PR is on top of #9079, adding the VS Code extensions and settings so that a developer can get IDE features out-of-box.

Test Plan:
---

[See this instructional video](https://www.facebook.com/yangbo.0/videos/294904826169431)

How to set up the VS Code based HHVM development environment from a fresh Ubuntu:

1. Install Nix and checkout HHVM
   ```
   sh <(curl -L https://nixos.org/nix/install) --daemon &&
   git clone git@github.com:Atry/hhvm.git --depth 1 --recurse-submodules --branch vs-code
   ```
2. Open `hhvm.code-workspace` added by this PR from VS Code
3. Install extension recommendations
4. Reload VS Code
5. Wait for the extensions to be activated. Note that it would take minutes to download dependencies, depending on your network speed.
6. Click the Build button from VS Code

You can also run `cmake` and other build tools from command line by using `nix-shell`.
